extensions/mssql/src/contracts.ts

```diff
@@ -36,19 +36,48 @@ export interface RequestSecurityTokenParams {
 	authority: string;
 	provider: string;
 	resource: string;
+	accountId: string;
```

Comment:
Add JS doc comments for all properties you're adding (it would be great if you could add them for the existing properties on these classes as well but to start with let's at least just add them for the new ones you're adding)